develop branch fails to start when using graphdb

The latest `develop` branch (2b9f9d9) can run if we use the `development` spring profile with the in-memory triple store.

However, if we set up and configure a GraphDB triple store instead, the FDP crashes:

```none
2025-02-21 14:18:01,073 732  [main] INFO  org.fairdatapoint.Application - Starting Application using Java 21.0.5 with PID 72890 (/home/me/Documents/repos/fdp/target/classes started by me in /home/me/Documents/repos/fdp)
2025-02-21 14:18:01,075 734  [main] INFO  org.fairdatapoint.Application - The following 1 profile is active: "development"
...
Error starting ApplicationContext. To display the condition evaluation report re-run your application with 'debug' enabled.
2025-02-21 14:18:06,271 5930 [main] ERROR org.springframework.boot.SpringApplication - Application run failed
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'rdfDevelopmentMigrationRunner': Invocation of init method failed
...
Caused by: org.fairdatapoint.entity.exception.ValidationException: Parent is not of type (missing type: https://w3id.org/fdp/fdp-o#MetadataService)
...
Process finished with exit code 1
```

Additional configuration:

```yaml
spring:
  autoconfigure:
    exclude: org.springframework.boot.autoconfigure.mongo.MongoAutoConfiguration

repository:
  main:
    type: 4
    graphDb:
      url: http://localhost:7200
      repository: fdp-main
  drafts:
    type: 4
    graphDb:
      url: http://localhost:7200
      repository: fdp-drafts
```

The `fdp-main` and `fdp-drafts` repos were created manually using the graphdb console, using graphdb 10.8.1.

--- a/src/main/java/org/fairdatapoint/config/RepositoryConfig.java
+++ b/src/main/java/org/fairdatapoint/config/RepositoryConfig.java
@@ -59,7 +59,7 @@ public class RepositoryConfig {
 
     @Bean(initMethod = "init", destroyMethod = "shutDown", name = "draftsRepository")
     public Repository draftsRepository(ApplicationContext context) throws RepositoryException {
-        return prepareRepository(context, repositoryProperties.getMain());
+        return prepareRepository(context, repositoryProperties.getDrafts());
     }
 
     public Repository prepareRepository(ApplicationContext context, RepositoryConnectionProperties properties)

